BEIMA.Backend.Test/MongoService/MongoConnectorTest.cs

```diff
@@ -497,6 +497,55 @@ public void ConnectorCreated_UpdateNullBuilding_NullReturned()
             Assert.IsNull(result);
         }
 
+        [TestCase("name", "a12345")]
+        [TestCase("key1", 12345)]
+        [TestCase("key2", true)]
+        public void InsertBuilding_GetFilteredBuildings_BuildingInList(string key, dynamic value)
```

Comment:
I'm guessing that these are just changes from branching off of your building branch. I would target that branch instead of main for now so that we can see only the relevant changes. Then update it to target main once the building changes merge it.

Reply:
That is correct. My bad, I meant to target the other branch instead of main.